# Semantic error: break statement outside a loop
fn main() -> int
{
  break;
}
